packages/graphql/src/gen/nxs.gen.ts

```diff
@@ -155,6 +155,7 @@ export interface NexusGenFieldTypes {
     title: string; // String!
   }
   Mutation: { // field return type
+    addProject: NexusGenRootTypes['App'] | null; // App
```

Comment:
I don't think App is supposed to be nullable, is it? Might be good to find a way to make it non nullable (not sure the nexus syntax off the top of my head).

Reply:
I agree, I'll look at some of the other ones and see what is the syntax for making it non nullable.

Reply:
While looking at this more I noticed all the mutations returned their type or null. I'm not sure if that's correct but I'll keep digging into this

Reply:
Changing `t.field` to `t.nonNull.field` will change this